### ISSUE TITLE
Minor Suggested Change To Installing Documentation

### ISSUE THREAD

**Issue by EvilSupahFly** [overviewer/Minecraft-Overviewer#2094]
Since Ubuntu 20.04, Ubuntu and anything derived from it (like Mint), the use of `apt-key` has been deprecated. Officially, users should manage keyring files in `/etc/apt/trusted.gpg.d` instead. Although `apt-key` still technically works as of "Jammy", there's no guarantee it will continue to do so as Debian claimed it would be given the Red Shirt treatment in April of 2022.

There is a work-around published on [Ask Ubuntu](https://askubuntu.com/questions/1328806/how-to-solve-apt-key-deprecated), if anybody wants to know the long way around, but I think it would be easier (and faster) to follow in the footsteps of Wine HQ, as [their documentation](https://wiki.winehq.org/Ubuntu) has been updated accordingly.

It's not super urgent, but it's probably better to address this before `apt-key` is removed from production entirely and you get a slew of tickets for it.

**Comment by ajh123** [comment]
The main README states ..
> PLEASE NOTE: Overviewer is currently unmaintained. PRs will not be merged and issues will not be addressed. The website and repository will remain online and accessible.
Instead please use the new working (“successor”) version of Overviewer supporting 1.20 worlds and textures.
https://github.com/GregoryAM-SP/The-Minecraft-Overviewer

**Comment by Gregory-AM** [comment]
Sadly, at the moment, I have not implemented any fixes towards Linux or Mac Machines.

I'm unsure as to this issue, or anything related to it as I'm a Windows Programmer.
But, since I do have a Chromebook, I will attempt to see what this issue may be, and see if there is anything I can do to fix it.

**Comment by EvilSupahFly** [comment]
> Sadly, at the moment, I have not implemented any fixes towards Linux or Mac Machines.
> 
> I'm unsure as to this issue, or anything related to it as I'm a Windows Programmer. But, since I do have a Chromebook, I will attempt to see what this issue may be, and see if there is anything I can do to fix it.

For testing purposes, I would suggest a VM, like Oracle's since Google locks down Chrome OS implementations, unless you're handy enough to bypass those measures.

**Comment by ajh123** [comment]
> > Sadly, at the moment, I have not implemented any fixes towards Linux or Mac Machines.
> > I'm unsure as to this issue, or anything related to it as I'm a Windows Programmer. But, since I do have a Chromebook, I will attempt to see what this issue may be, and see if there is anything I can do to fix it.
> 
> For testing purposes, I would suggest a VM, like Oracle's since Google locks down Chrome OS implementations, unless you're handy enough to bypass those measures, which is often more work than it's worth for what you get back in functionality.

Or as I suggested in #2087 use Windows Subsystem for Linux

**Comment by Gregory-AM** [comment]
@ajh123, Agreed. I can do this, the reason I wanted a second SSD was specifically for a Linux install since it's a good OS to use for development.

But, I'll be checking out the option to have a VM on Windows using the methods you've mentioned.
